src/components/Layout/AppLayout/AppLayout.vue

```diff
@@ -209,9 +257,18 @@ export default {
     window.addEventListener('resize', this.handleResize);
     this.$nextTick(() => {
       const { header } = this.$refs;
-
-      this.headerSize = header ? header.offsetHeight : 0;
+      this.headerSizeHeight = header ? header.offsetHeight : 0;
+      this.$nextTick(() => {
+        setTimeout(() => {
+          const { title, subtitle, headline } = this.$refs;
+
+          this.titleSizeWidth = title.offsetWidth < title.scrollWidth;
+          this.subtitleSizeWidth = subtitle.offsetWidth < subtitle.scrollWidth;
+          this.headLineSizeWidth = headline.offsetWidth < headline.scrollWidth;
+        }, 5000);
+      });
```

Comment:
**note:** How does this 5 seconds delay timeout work? Can u record a video on it?

Reply:
This delay is just for the component to update correctly when it is waiting for the sidebar content that comes through requests. This delay is almost imperceptible.